### PR TITLE
container_status: Add debug info for container

### LINKED ISSUE #1752: Support returning the container pid in the ContainerStatus info map
I'd love to be able to retrieve the process pid from the ContainerStatus to be able to correlate containers with a process tree. This is to support tooling such as [Scope](https://github.com/weaveworks/scope).

Containerd has something along those lines: https://github.com/containerd/cri/blob/master/pkg/server/container_status.go#L102.

Now, the CRI interface doesn't seem to define any well-known keys and values for this info map:
```
message ContainerStatusResponse {
    // Status of the container.
    ContainerStatus status = 1;
    // Info is extra information of the Container. The key could be arbitrary string, and
    // value should be in json format. The information could include anything useful for
    // debug, e.g. pid for linux container based container runtime.
    // It should only be returned non-empty when Verbose is true.
    map<string, string> info = 2;
}
```
It'd be beneficial to standardise/define a bit of this. From what I gather from the containerd implementation, there's an `"info"` key with a serialized JSON object as value. That JSON object contains the `"pid"` key, along with a lot more details.

### PR DESCRIPTION
**Pull request by mrunalp**
Add pid and sandboxId for container

Fixes #1752.

Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by runcom** [comment]
LGTM

File: server/container_status.go [hunk 1]
```diff
@@ -101,6 +102,11 @@ func (s *Server) ContainerStatus(ctx context.Context, req *pb.ContainerStatusReq
 	resp.Status.State = rStatus
 	resp.Status.LogPath = c.LogPath()
 
+	resp.Info = map[string]string{
```
Comment (runcom) [inline server/container_status.go]: We should check if `Verbose` is `true` in the request before adding this as per:
```
 2461     // It should only be returned non-empty when Verbose is true.
```

Comment (mrunalp) [reply server/container_status.go]: Fixed.

**Comment by mrunalp** [comment]
We can get this in after #1763

**Pull request merged**